kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/Environment.java

```diff
@@ -140,12 +144,12 @@ private static <T> T getOrDefault(String varName, Function<String, T> converter,
     }
 
     private static String readMetadataProperty(String property) {
-        var p = new Properties();
+        Properties p = new Properties();
```

Comment:
is there a reason to move away from `var` to explicitly typed variables?

Reply:
It was failing locally DKW and changing that Intellij doesn't complain anymore.

Reply:
Double check your jdk version/language settings in idea as `var` should be usable